pkg/genericapiserver/server/openapi/openapi.go

```diff
@@ -33,13 +33,15 @@ import (
 
 const (
 	OpenAPIVersion = "2.0"
+	ExtensionTags  = "x-kubernetes-tags"
```

Comment:
Create a specific name for each extension, rather than a generic one.  I don't think this name describes what we're exposing well